### PR TITLE
Fix S2I .NET documentation for Online

### PR DESCRIPTION
**Pull request by Miciah**
• Minor fixes ("high-quality applications", delete a stray comma, use singular "registry" where appropriate).

• Rework the documentation around image streams:

  • For Online, simply provide the name of the image stream.

  • For Enterprise, state that hte image stream is created on installation.

  • For Origin and Dedicated, state that the image streams need to be created.

• For Online, use the image stream in commands.

• For Online, ifdef out the entire section on templates since the templates are not provided in Online.

---

FYI @ahardin-rh.

### CODE REVIEW

File: using_images/s2i_images/dot_net_core.adoc [hunk 1]
```diff
@@ -38,30 +38,38 @@ and Python from within {product-title}.
...
+The RHEL 7 images are available through Red Hat's subscription registry:
 
 ----
 $ docker pull registry.access.redhat.com/dotnet/dotnetcore-10-rhel7
 $ docker pull registry.access.redhat.com/dotnet/dotnetcore-11-rhel7
 ----
 
-To use these images, you can either access them directly from these
+ifdef::openshift-online[]
+You can use these images through the "dotnet" image stream.
```
Comment (ahardin-rh) [inline using_images/s2i_images/dot_net_core.adoc]: `dotnet`

File: using_images/s2i_images/dot_net_core.adoc [hunk 2]
```diff
@@ -38,30 +38,38 @@ and Python from within {product-title}.
...
 
 [[dot-net-core-installing-images]]
 == Images
 
-Image stream definitions for the .NET Core on RHEL S2I image are now added
-during {product-title} installations.
-endif::openshift-enterprise[]
-
-The RHEL 7 images are available through Red Hat's subscription registry using:
+The RHEL 7 images are available through Red Hat's subscription registry:
```
Comment (ahardin-rh) [inline using_images/s2i_images/dot_net_core.adoc]: s/Red Hat's subscription registry/the Red Hat Registry

File: using_images/s2i_images/dot_net_core.adoc (hunk 1 above)
Comment (Miciah) [inline using_images/s2i_images/dot_net_core.adoc]: Fixed.  Thanks!

File: using_images/s2i_images/dot_net_core.adoc (hunk 2 above)
Comment (Miciah) [inline using_images/s2i_images/dot_net_core.adoc]: Fixed.  Thanks!

**Comment by ahardin-rh** [comment]
@Miciah Thank you!!!!

**Comment by ahardin-rh** [comment]
No revision history needed

**Pull request merged**